### PR TITLE
Marked problematic test with "Skip" attribute

### PR DESCRIPTION
**Pull request by Danil-Andrianov**
Froze #6501

## Proposed changes

- Problematic test  in _main_ `System.Windows.Forms.UITests.PrintPreviewDialogTests`
is ignored for now.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6504)

### CODE REVIEW

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public async Task PrintPreviewDialog_Hotkey_Ctrl_4Async()
             });
         }
 
-        [WinFormsFact]
+        [WinFormsFact(Skip = "For some reasons that test can't pass pipelines while locally it works perfectly fine.")]
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs:74]: Do you have an issue to track this and fix? Please add as much details as you know on why this is failing. please link the issue here and i can merge this PR after that.

Comment (Danil-Andrianov) [reply src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs:74]: Sorry, there was a typo in the PR description and tracking issue was not created. It's fixed now.

**Review by Tanya-Solyanik — APPROVED** [review]

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs [hunk 2]
```diff
@@ -20,6 +20,7 @@ public async Task PrintPreviewDialog_Hotkey_Ctrl_1Async()
         {
             await RunTestAsync(async printPreviewDialog =>
             {
+                printPreviewDialog.Focus();
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs]: Why is this necessary? The dialog is in focus by the virtue of the app start.

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs [hunk 3]
```diff
@@ -10,6 +10,9 @@ namespace System.Windows.Forms.UITests
 {
     public class PrintPreviewDialogTests : ControlTestBase
     {
+        // This value may need to be adjusted if tests fail in CI/different environment.
+        private const int DelayMS = 100;
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs:14]: We have no windows events here to confirm?

Comment (RussKie) [reply src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/PrintPreviewDialogTests.cs:14]: That's an interesting angle. Maybe?..

**Comment by gpetrou** [comment]
In `SendAsync`, shouldn't `var foreground = GetForegroundWindow();` be `foregroundWindow = GetForegroundWindow();`?

**Comment by dreddy-work** [comment]
@Danil-Andrianov, can we get this in ASAP to avoid noise in CI?  please resolve pending comments.
